lib/pengine/utils.c

```diff
@@ -1077,6 +1077,34 @@ unpack_operation(pe_action_t * action, xmlNode * xml_obj, pe_resource_t * contai
         g_hash_table_remove(action->meta, field);
     }
 
+    /*
+     * Timeout order of precedence:
```

Comment:
The override order is easier to follow now thanks to the commits before this one. I added the precedence table to reduce the likelihood of regressions in the future, and to make it easy to see at a glance how the timeout should be chosen. I'm open to removing this table or putting it elsewhere.